transports/websocket/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use libp2p_core::{
     transport::{map::MapFuture, ListenerId, TransportError, TransportEvent},
     Transport,
 };
-use rw_stream_sink::RwStreamSink;
+pub use rw_stream_sink::RwStreamSink;
```

Comment:
is it okay to reexport to avoid importing `rw_stream_sink` in `src/bandwidth.rs`?

Reply:
I'd suggest make a type alias for the stream. That is why I did for TCP.